src/test/java/org/futurecollars/lesson2task4ageVeryfication/AgeVeryficationTest.java

```diff
@@ -0,0 +1,27 @@
+package org.futurecollars.lesson2task4ageVeryfication;
+
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class AgeVeryficationTest {
+
+    @Test
+    public void shouldVerifyAge(){
```

Comment:
W jednym teście sprawdzasz trzy różne przypadki. Rozbij to na osobne testy i będzie super